lesson-3/coalition/src/setupTests.js

```diff
@@ -0,0 +1,4 @@
+// http://airbnb.io/enzyme/docs/installation/index.html#working-with-react-16
+import { configure } from 'enzyme'
+import Adapter from 'enzyme-adapter-react-16'
+configure({ adapter: new Adapter() })
```

Comment:
found out during course this could be moved to git root to `npm test` without `cd src`

Reply:
aah, just one dir up, not git root in this multi lesson repo :)